### PR TITLE
[IIIF-1111] Upgrade to Cantaloupe v5

### PR DESCRIPTION
**Pull request by ksclarke**
* Upgrade to Cantaloupe v5
* Remove now unused Imagemagick/Graphicsmagick libraries
* Make libturbojpeg the default jpeg processor
* Install Grok JPEG-2000 library
* Make Grok the default JP2 library if Kakadu isn't installed
* Add `docker.noCache` to the version check profile
* Update container/OS package versions to the latest versions
* Remove wget from the container since it wasn't being used
* Add new config files for Cantaloupe v5 (comments kept this time)
* Remove Fluency stuff now that upstream supports Logstash
* Added default log file locations, but still kept default log setting to off

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,29 +64,28 @@
...
-    <openjdk.version>11.0.10+9-0ubuntu1~20.04</openjdk.version>
+    <openjdk.version>11.0.11+9-0ubuntu2~20.04</openjdk.version>
     <gcc.version>4:9.3.0-1ubuntu2</gcc.version>
     <make.version>4.2.1-1.2</make.version>
     <libtiff.version>4.1.0+git191117-2ubuntu0.20.04.1</libtiff.version>
     <build.essential.version>12.8ubuntu1.1</build.essential.version>
-    <libopenjp2.version>2.3.1-1ubuntu4</libopenjp2.version>
+    <libopenjp2.version>2.3.1-1ubuntu4.20.04.1</libopenjp2.version>
     <libturbojpeg.version>2.0.3-0ubuntu1.20.04.1</libturbojpeg.version>
     <wget.version>1.20.3-1ubuntu1</wget.version>
```
Comment (markmatney) [inline pom.xml:82]: one last straggling `wget.version` use

File: src/main/docker/configs/cantaloupe.properties.default-5.0.2 [hunk 2]
```diff
@@ -0,0 +1,753 @@
...
+CANTALOUPE_MANUAL_PROCESSOR_FLV = FfmpegProcessor
+CANTALOUPE_MANUAL_PROCESSOR_GIF =
+CANTALOUPE_MANUAL_PROCESSOR_JP2 = GrokProcessor
+CANTALOUPE_MANUAL_PROCESSOR_JPG =
+CANTALOUPE_MANUAL_PROCESSOR_MOV = FfmpegProcessor
+CANTALOUPE_MANUAL_PROCESSOR_MP4 = FfmpegProcessor
+CANTALOUPE_MANUAL_PROCESSOR_MPG = FfmpegProcessor
+CANTALOUPE_MANUAL_PROCESSOR_PDF = PdfBoxProcessor
+CANTALOUPE_MANUAL_PROCESSOR_PNG =
+CANTALOUPE_MANUAL_PROCESSOR_TIF = TurboJpegProcessor
```
Comment (markmatney) [inline src/main/docker/configs/cantaloupe.properties.default-5.0.2:334]: > Make libturbojpeg the default jpeg processor

Just to clarify for myself, is this the line that's doing this?

**Review by markmatney — CHANGES_REQUESTED** [review]
LGTM, just one property declaration in the POM that could be removed

File: pom.xml (hunk 1 above)
Comment (ksclarke) [reply pom.xml:82]: Thanks, I'll remove.

File: src/main/docker/configs/cantaloupe.properties.default-5.0.2 (hunk 2 above)
Comment (ksclarke) [reply src/main/docker/configs/cantaloupe.properties.default-5.0.2:334]: Yes, but as you're guessing I think that should be on the JPG config, not TIF.

**Review by markmatney — APPROVED** [review]

**Review by DRickard — APPROVED** [review]

**Pull request merged**